### DISCUSSION TITLE
I am looking for an internship opportunity to learn

### DISCUSSION THREAD

**Discussion by maggickkw** [community/community#55631] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I am kindly looking for an internship to learn more and develop my skills. I don't require to be paid. I want to only learn and grow as a developer !  

Some kindly assist me !

**Comment by AbdullahWins** [comment]
1. learn the basics of programming
2. create few small projects and also get used to version control systems like git
3. build some decent sized project that involves and solves real life issues.
4. use CRUD operations, learn APIs and Best practices.
5. finally apply for an internship with a proper portfolio and with some projects

**Reply by maggickkw** [reply]
I have done all 5 steps... Still not getting anything!

**Reply by AbdullahWins** [reply]
Hey, if you have already done these, please create a resume which contains the live links of your project and also mentions your skillsets.

I went through your github repos and found you didn't host any live site.
That's a huge mistake, nobody will download and install the project to find out what's inside.

I would advice you to host your sites live so that anyone can just click a link and see the masterpice you have created.

Here's some free hosting options:
1. github pages (for html)
2. netlify, vercel (for frontend)
3. render (for backend)

Hope it helps. Best of luck.

**Comment by queenofcorgis** [comment]
Hi @maggickkw , we're happy you're here 😀

However, we currently do not allow job listings or seeking on the Community on Discussions. It is something that we've added to the list to investigate, if we can do so without overwhelming other conversations. Keep checking back as the Community grows and we add more capabilities!

Thanks for your interest 👍
